sql/parser/builtins.go

```diff
@@ -214,7 +214,6 @@ var builtins = map[string][]builtin{
 		},
 	},
 
-	// TODO(XisiHuang): support the position(substring in string) syntax.
```

Comment:
It might be worth leaving a comment here explaining that the parser coerces "POSITION" to "STRPOS".